jq/script-05.sh

```diff
@@ -5,3 +5,11 @@ set -euo pipefail
 # The input for this script is the scores.json file.
 # TODO: Write a command to output the names of each player, as well as their city.
 # Your output should contain 6 lines, each with two words on it.
+jq -r '.[] | "\(.nam
+e) \(.city)"' scores.json
```

Comment:
How did you end up with a newline in the middle of your command here? (Same for script-06 and script-07, script-10, and script-11)?